erpnext/hooks.py

```diff
@@ -201,7 +201,7 @@
 	"Sales Invoice": {
 		'validate': 'erpnext.regional.india.utils.set_place_of_supply',
 		"on_submit": "erpnext.regional.france.utils.create_transaction_log",
-		"on_trash": "erpnext.regional.france.utils.check_deletion_permission"
+		"on_trash": ["erpnext.regional.france.utils.check_deletion_permission", "erpnext.regional.nepal.utils.check_deletion_permission"]
```

Comment:
@theopen-institute thanks for the PR!

just make a common function and just add to the list of countries.

`erpnext.regional.check_delete_permission`